src/TMG.Aimsun/AimsunControllerParameter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMG.Aimsun
+{
+    public class AimsunControllerParameter
+    {
+        internal string Name;
+        internal string Value;
+        public AimsunControllerParameter(string name, string value)
+        {
+            Name = name;
+            Value = value;
+        }
+    }
+}
```

Comment:
Is this class still used?  If not please remove.

Reply:
deleted